frontend/src/components/AppBetaAlert.tsx

```diff
@@ -0,0 +1,23 @@
+import { useTranslations } from "next-intl";
+
+import FullWidthAlert from "./FullWidthAlert";
+
+const BetaAlert = () => {
+  const t = useTranslations("Beta_alert");
+  const heading = t.rich("alert_title", {
```

Comment:
This file was added until #1361

Reply:
Should we add a comment in the code that points to that ticket? Format we've been using is `// TODO (Issue #1361): ...`